### PR TITLE
Switch Friday BOT to use Plugins

### PR DESCRIPTION
**Pull request by huan**
1. use RoomInvitor & VoteOut plugin

### CODE REVIEW

File: src/plugins/vote-out.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/* eslint-disable sort-keys */
+import {
+  VoteOut,
+  VoteOutConfig,
+}                 from 'wechaty-voteout'
+
+// import {
+//   DEVELOPERS_ROOM_ID_LIST,
+// }                           from '../rooms-config'
```
Comment (windmemory) [inline src/plugins/vote-out.ts:9]: Shall we remove these comments? Is there any particular reason to keep this?

File: src/config.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import readPkgUp from 'read-pkg-up'
 
 import {
-  KeywordRoomConfig,
+  // KeywordRoomConfig,
```
Comment (windmemory) [inline src/config.ts:4]: Shall we remove this?

File: src/plugins/vote-out.ts (hunk 1 above)
Comment (huan) [reply src/plugins/vote-out.ts:9]: Thanks for the review!

The comments are not removed (yet) is because that this PR is still working on progress...

Comment (huan) [reply src/plugins/vote-out.ts:9]: I should create this PR with the `DRAFT` however I forgot.

Will continue to push commits tonight.

**Comment by huan** [comment]
@windmemory I'm working on refactoring our `VoteManager` to @Gcaufy 's `VoteOut` plugin.

I'll try to merge your VoteManager into VoteOut later, please feel free to send PR if I missed anything.

**Pull request merged**